src/CoreBundle/ChameleonHttpKernel.php

```diff
@@ -53,23 +37,10 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
                 }
             }
             if (count($aTrustedProxies) > 0) {
-                Request::setTrustedProxies($aTrustedProxies);
+                Request::setTrustedProxies($aTrustedProxies, Request::getTrustedHeaderSet());
```

Comment:
Request::getTrustedHeaderSet() is never set to any value?

Reply:
Yes it is. Any service can set it and if it hasn't been set, yet, it is the default. Because this is a mandatory parameter and we do not want to change it here, we just provide the current state.